tests/simpleharness/GetAllocatedBytesForCurrentThreadTest.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.Xunit.Performance;
+using System;
+using System.Reflection;
+
+namespace simpleharness
+{
+    public class GetAllocatedBytesForCurrentThreadTest
+    {
+        [Benchmark]
+        public static void WithCollect()
+        {
+            var method = typeof(GC)
+                .GetTypeInfo()
```

Comment:
Yup, we talked about this offline as well.  Sounds good.  I think we should fix it up in a subsequent PR so that we have "examples" and "tests".